### PR TITLE
swapped out web3modal for connectkit

### PR DESCRIPTION
**Pull request by memosys**
- fixes signout on network change
- no more connect issues

### CODE REVIEW

File: web-portal/frontend/utils/Web3Provider.tsx [hunk 1]
```diff
@@ -17,39 +17,44 @@ Object.entries(envVariables).forEach(([key, value]) => {
...
+export const config = createConfig(
+  getDefaultConfig({
+    chains: [mainnet, base, gnosis, optimism],
+    transports: {
+      [mainnet.id]: http(
+          `https://eth-mainnet.rpc.grove.city/v1/1eec3963`
+      ),
+      [base.id]: http(`https://base-mainnet.rpc.grove.city/v1/1eec3963`),
+      [gnosis.id]: http(`https://gnosischain-mainnet.rpc.grove.city/v1/1eec3963`),
+      [optimism.id]: http(`https://optimism-mainnet.rpc.grove.city/v1/1eec3963`),
```
Comment (plor) [inline web-portal/frontend/utils/Web3Provider.tsx:32]: Do you mean to be committing these?

**Review by plor — COMMENTED** [review]
Good, but not sure about the possible secrets

File: web-portal/frontend/utils/Web3Provider.tsx (hunk 1 above)
Comment (memosys) [reply web-portal/frontend/utils/Web3Provider.tsx:32]: no regrets! will be disabling!

Comment (memosys) [reply web-portal/frontend/utils/Web3Provider.tsx:32]: we need to swap to own endpoints at some point

**Pull request merged**